### DISCUSSION TITLE
Can I suppress PR creation base on release info?

### DISCUSSION THREAD

**Discussion by zmes50416** [renovatebot/renovate#13297] [Request Help] [answered]
I want to not have PR created before certain criteria(e.x. at least 30% Adoption rate or high merge Confidence). 
After looking at the document, I found that `internalChecksFilter` looks like what I want but only with stabilityDays?

Is there another way to do it? Thank

**Comment by rarkins** [comment, marked as answer]
This is only possible for stabilityDays currently. e.g. set stabilityDays=7 and prCreation=not-pending
